english/little-me/Apple/Apple.md

```diff
@@ -0,0 +1,10 @@
+Thank you so much.
+Now that I am full I can rewind the time.
+
+Do you like to see my powers?
+
+You Choosed To Say:
+
+[Yes](https://github.com/udacity/create-your-own-adventure/tree/master/english/little-me/little-me.md)
+
+[No](https://github.com/udacity/create-your-own-adventure/tree/master/english/)
```

Comment:
The story ends once the user clicks No so I thought maybe it's a good idea
to redirect the reader to the main English page.